configs/tasks/rearrange/pick.yaml

```diff
@@ -44,52 +51,52 @@ TASK:
         TYPE: "IsHoldingSensor"
     RELATIVE_RESTING_POS_SENSOR:
         TYPE: "RelativeRestingPositionSensor"
-    LOCAL_SENSOR:
-        TYPE: "LocalizationSensor"
-    SENSORS: ["TARGET_START_SENSOR", "GOAL_SENSOR", "JOINT_SENSOR", "IS_HOLDING_SENSOR", "END_EFFECTOR_SENSOR", "LOCAL_SENSOR", "RELATIVE_RESTING_POS_SENSOR"]
+    SENSORS: ["TARGET_START_SENSOR", "JOINT_SENSOR", "IS_HOLDING_SENSOR", "END_EFFECTOR_SENSOR", "RELATIVE_RESTING_POS_SENSOR"]
     ROBOT_FORCE:
         TYPE: "RobotForce"
         MIN_FORCE: 20.0
+    FORCE_TERMINATE:
```

Comment:
I think this one can be misinterpreted: Is this a "force terminate" in the sense of "forcing a process to terminate" or is it in the sense that if the force (applied on the Agent) is too high the agent terminates? f it is the latter, I think this should have the same pattern as "CONSTRAINT_VIOLATE_SHOULD_END". Something like "EXCESSIVE_FORCE_SHOULD_END".